packages/account-sdk/tsconfig.build.json

```diff
@@ -1,10 +1,8 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "allowJs": true,
-    "declaration": true,
-    "declarationMap": true,
-    "sourceMap": true
+    "allowJs": false,
```

Comment:
we removed the only JS code from vendor during walletlink removal, flipping this flag